docs/contributing/contributing.md

```diff
@@ -1,111 +1,168 @@
+:::info
+Contributed content for Palo Alto Networks products (e.g., PAN-OS, XDR, etc.) will be adopted and supported by the Cortex XSOAR Content team.  
+:::
 
-**Note:** If you plan to contribute content that integrate with one of Palo Alto Networks products, we will require the pack to be supported by Cortex XSOAR and therfore we might adopt your pack and push it into the Cortex XSOAR development pipeline to be released as Cortex XSOAR supported.
+## Contribution Methods
+After creating new content to contribute, you must submit your content to Palo Alto Networks. The Cortex XSOAR Content Team will review and approve it before it becomes available to customers.
+Before starting to work on a contribution, it is recommended to go over the [Contribution SLA](../contributing/sla) article.
 
-For more information about the different support levels, check the [official documentation](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/marketplace/content-packs-support-types.html).
+**There are several ways to submit contributions:**
 
-## What to contribute
-If you’re looking for an easy way to start contributing Content to Cortex XSOAR Marketplace, have a look at the issues labeled as [good first issue](https://github.com/demisto/content/labels/good%20first%20issue), which are low-hanging fruits.
+### Cortex XSOAR Content GitHub Repository (Pull Request)
+Contribute by creating a fork and opening a pull request on the [Cortex XSOAR Content GitHub repository](https://github.com/demisto/content).  
+Use this method in the following scenarios:
+* You are a Technology Partner contributing with *partner* supported new content.
+* Your contribution is big and contains lots of different parts (integrations, scripts, playbooks, layouts, etc.) that is likely to lead to a complex review process.
+* You are proficient with GitHub.
 
-These issues are particularly well suited for external contributions. If you decide to start on an issue, [assign yourself](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users#assigning-an-individual-issue-or-pull-request) so other contributors from the community know that you're working on it.
-## How to contribute
+:::tip
+For contributing through GitHub, we recommend to use a GitHub Codespace, which will provide you with a pre-configured ready-to-use development environment.  
+This method is still experimental, but will make the contribution, development, and review processes much easier.  
 
-After you have created your content, you must submit your content to Palo Alto Networks: the Cortex XSOAR Content Team will review and approve it before it becomes available to customers.
+For more information and a step-by-step guide, see the dedicated [GitHub Codespace Setup tutorial](../tutorials/tut-setup-dev-codespace.md).
+:::
```

Comment:
Wouldn't a link for the "[Set Up Your Dev Environment](https://xsoar-pan-dev--pull-request-1336-0i25g2kq.web.app/docs/tutorials/tut-setup-dev)" doc would be more fitting (instead of the first one)?